app/src/main/res/layout/fragment_home.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/purple_200"
+    tools:context=".HomeFragment">
+
+    <TextView
+        android:id="@+id/tvHomeFragment"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Home"
```

Comment:
Make more use of the string.xml ressource, this way if you have to change a value, you only need to change it once